paper/src/main/java/vg/civcraft/mc/citadel/activity/XZKey.java

```diff
@@ -0,0 +1,23 @@
+package vg.civcraft.mc.citadel.activity;
+
+class XZKey {
```

Comment:
Can use a record

Reply:
Good point, I'll look it later today

Reply:
Done: https://github.com/CivMC/Citadel/pull/39/commits/b83610d297d9730adf41d1a3bafb6d8ddfdd4de1#diff-4c1137a79e09af297c7f1cc3ecac9a852e159e840799a1565fcdaa586bbcb522R3